Brady

meretribia opened the pull request:
This branch has Bailey's schema changes, Francesco's data factory, and my own tweaks to keep it deploy-ready.  Check update notes.

bdawk28 committed:
update ERD

Commit:
add data factory to project

Commit:
removed picklist values and matched with destination org

Commit:
add more record type attributes

Commit:
persist changes

Commit:
add test class for datafactory

Commit:
fixed datafactory test class for 100% coverage

Commit:
relate objects

meretribia committed:
Changed to allow deployment in my own org

Tweaked record types and so on (probably undid some changes) using existing metadata, then renamed Appointment fields and set sharing model for Appointment to ControlledByParent.  This all together allowed me to deploy without complaints.

meretribia committed:
Update package.xml

Add a comment to package.xml

meretribia committed:
Big update with all the metadata from teambig org and Francesco's factory

Making a new branch for this update which I will then merge with my own name so I stop littering the repo.

meretribia committed:
Update Brady with org metadata

Updating my branch with all metadata from the most current incarnation of our org.  This should deploy anywhere.

FrancescoWard reviewed (approved):
Looks good to me. Great work !

meretribia committed:
Modified data factory

Added a few comments, removed AppointmentContacts list, and changed Account object so it was always referencing Southern Heart.

Commit:
more complete datafactory

meretribia committed:
Account patch 2

Added in a try-catch block for a queryexception and updated the test class to keep 100% coverage.

meretribia committed:
Merge branch 'francesco-datafactory-w-test' into brady

meretribia committed:
Modified schema and data factory

I changed the factory so the appointment-contact and resident-doctor adding makes slightly more sense.  Deleted AppointmentContact from org and added Residence field to Appointment.

bdawk28 reviewed (approved):
ok boss
